pajbot/modules/slotmachine.py

```diff
@@ -24,7 +24,7 @@ def pull_lol(low_tier_emotes, high_tier_emotes, bet, house_edge, ltsw, htsw, ltb
     for e in high_tier_emotes:
         slot_options += [e]
 
-    randomized_emotes = random.choice(slot_options, 3).tolist()
+    randomized_emotes = random.sample(slot_options, 3)
```

Comment:
should be `random.choices(slot_options, k=3)` as random.sample returns a unique list of choices

see https://docs.python.org/3/library/random.html#random.choices and https://docs.python.org/3/library/random.html#random.sample